executor/builder.go

```diff
@@ -930,14 +930,15 @@ func (b *executorBuilder) buildLoadData(v *plannercore.LoadData) Executor {
 		return nil
 	}
 
-	worker, err := NewLoadDataWorker(b.ctx, v, tbl)
+	base := newBaseExecutor(b.ctx, nil, v.ID())
```

Comment:
Why advance the initialization of `base`?

Reply:
I want to let LoadDataWorker use getSysSession/releaseSysSession of it.